core/domain/feedback_services_test.py

```diff
@@ -87,11 +86,13 @@ class SuggestionQueriesUnitTests(test_utils.GenericTestBase):
     def _generate_thread_id(self, unused_exp_id):
         return self.THREAD_ID1
 
-    def _get_threads(self, unused_exploration_id):
-        return self.threads
+    def _put_threads(self):
```

Comment:
Ah -- I was wondering why you put _put_threads() everywhere instead of putting it in setUp(). I guessed that maybe it was because it would cause problems with test_create_and_get_suggestion() -- hence my last comment. However, if it doesn't, it's fine to leave it as-is!